### PR TITLE
S101 webpack

### PR DESCRIPTION
**Pull request by HadarRose**
Added webpack to project. Includes creating /dist folder for distribution files, package.json, webpack.config.js, and updating the README file

### CODE REVIEW

File: src/js/basicRequests.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
 const DEFAULT_IMAGE = "http://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png"; // the default image for twitter
 
-function getTimeline() {
-    console.info('Requesting timeline from backend.');
+getTimeline = function() {
```
Comment (kev-chan) [inline src/js/basicRequests.js]: don't use global scope

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/js/basicRequests.js (hunk 1 above)
Comment (kev-chan) [reply src/js/basicRequests.js]: global scope should only be used when absolutely needed

**Comment by kev-chan** [comment]
looks like you've still got old commits on this branch compared to master

**Comment by HadarRose** [comment]
@kev-chan I just saw that I indeed forgot to merge the old PR. Will take care of that now

File: src/js/main.js [hunk 2]
```diff
@@ -1,35 +1,9 @@
...
-        res.end();
-    });
-});
-
-http.createServer(handleRequest).listen(port);
+import '../css/index.css';
+import { getTimeline } from './basicRequests.js';
+
+document.addEventListener("DOMContentLoaded", ()=>{
+    document.getElementById("main-container").onload = getTimeline();
```
Comment (kev-chan) [inline src/js/main.js]: is attaching it to the onload needed?

Comment (HadarRose) [reply src/js/main.js]: Didn't think about that! just pushed a fix

**Comment by kev-chan** [comment]
The old commits should not be showing up in this PR and should only contain webpack relevant changes.  Please fix.  This matter down the road if you were to cherry-pick fixes from a squashed merge in production so you want keep branches specific to the proper fixes

**Review by kev-chan — APPROVED** [review]

**Pull request merged**